helpers/map-deep-merge.js

```diff
@@ -0,0 +1,223 @@
+//	___________        .___
```

Comment:
Please remove the `Todo` ASCI art.  The `TODO:` is enough so most IDEs will highlight it.